config/webpack/production.js

```diff
@@ -4,14 +4,14 @@ const webpack = require('webpack');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const ManifestPlugin = require('webpack-manifest-plugin');
 const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
 
 const optimization = Object.assign({}, {
   minimizer: [
-    new UglifyJsPlugin({
-      uglifyOptions: {
+    new TerserPlugin({
+      terserOptions: {
         parse: {
-          ecma: 8,
+          ecma: 2017,
```

Comment:
I believe I've interpreted this correctly according to this:

https://github.com/terser/terser#parse-options

> ecma (default: 2017) -- specify one of 5, 2015, 2016 or 2017. Note: this setting is not presently enforced except for ES8 optional trailing commas in function parameter lists and calls with ecma 2017.